### DISCUSSION TITLE
Channel 5

### DISCUSSION THREAD

**Discussion by ComeOnBear** [iptv-org/community#812] [Q&A]
### Channel ID

Channel5.uk

### Stream URL

unfortuntely not 

### Notes

British channel. listed in all the databases and the epg files also. 
i have no clue why they aren't in the index, probably but perhaps it is because there aren't any streams.
if 5STAR can also be added that would be great (i will create a new request if this violates the CONTRIBUTING.md)
apologies if this is a repeat thread, I haven't been able to find any posts.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Used to have a look on the C5 network for some URLs, sadly they're DRMd.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19780
